modules/about-the-single-model-serving-platform.adoc

```diff
@@ -30,6 +30,7 @@ When you have installed KServe, you can use the {productname-short} dashboard to
 
 * *TGIS Standalone ServingRuntime for KServe*: A runtime for serving TGI-enabled models
 * *Caikit-TGIS ServingRuntime for KServe*: A composite runtime for serving models in the Caikit format
+* *Caikit Standalone ServingRuntime for KServe*: A runtime for serving models in the Caikit Embeddings format for embeddings tasks
```

Comment:
Probably don't need to capitalize 'Embeddings' here. Compare with line 44, for example.